be/src/vec/exec/vbroker_scanner.cpp

```diff
@@ -91,9 +91,10 @@ Status VBrokerScanner::_fill_dest_columns(const Slice& line,
         return Status::OK();
     }
 
-    if (!check_array_format(_split_values)) {
-        return Status::OK();
-    }
+    // This check is meaningless, should be removed
```

Comment:
If we remove this, may be we should also covert the invalid array value in this row to null. Keep the same logic as cast function.